code/index.html

```diff
@@ -3,62 +3,99 @@
-        <optgroup label="hair color">
-          <option value="brown">brown hair</option>
-          <option value="yellow">yellow hair</option>
-          <!-- Add the rest of hair colors as options  -->
-        </optgroup>
-        <optgroup label="eye color">
-          <option value="hidden">hidden eyes</option>
-          <!-- Add the rest of eye colors as options  -->
-        </optgroup>
-        <optgroup label="accessories">
-          <option value="glasses">glasses</option>
-          <!-- Add the other accessory option here. (hat)  -->
-        </optgroup>
-        <optgroup label="other">
-          <option value="smoker">a smoking habit</option>
-        </optgroup>
-      </select>
+    <!-- side section containing questions, info & game setup -->
+    <aside class="side-section">
+      <img class="icon" src="images/guess-who-bubble.png" alt="Guess Who" />
+      <!-- New game setup (shown only on first page load) -->
+      <div class="side-section__content" id="newGame">
+        <h1 class="title">Welcome!</h1>
+        <input type="text" class="input-name" id="playerName" placeholder="What is your name?" />
+        <button id="startGame" class="button button__filled">Start Game</button>
+      </div>
 
-      <button id="filter" class="filled-button">FIND OUT</button>
+      <!-- Main game interaction content (hidden by default; visible after start game) -->
+      <div class="side-section__content hidden" id="questionSection">
+        <h1 class="title">Does the person have</h1>
+        <!-- Questions select input -->
+        <select class="select-questions" id="questions">
+          <optgroup label="hair color">
+            <option value="brown">brown hair</option>
+            <option value="yellow">yellow hair</option>
+            <option value="white">white hair</option>
+            <option value="grey">grey hair</option>
+            <option value="purple">purple hair</option>
+            <option value="black">black hair</option>
+            <option value="orange">orange hair</option>
+            <option value="hidden">hidden hair</option>
+          </optgroup>
+          <optgroup label="eye color">
+            <option value="blue">blue eyes</option>
+            <option value="brown">brown eyes</option>
+            <option value="green">green eyes</option>
+            <option value="hidden">hidden eyes</option>
+          </optgroup>
+          <optgroup label="accessories">
+            <option value="glasses">glasses</option>
+            <option value="necklace">a necklace</option>
+            <option value="earrings">earrings</option>
+            <option value="headCover">a head covering</option>
+          </optgroup>
+          <optgroup label="other">
+            <option value="smoker">a smoking habit</option>
+            <option value="facialHair">facial hair</option>
+            <option value="buttonShirt">a buttoned up shirt</option>
+          </optgroup>
+        </select>
```

Comment:
Maybe it would be a nice think to generate long lists of options with your Javascript? It is nothing wrong, with this approach, but It makes you type a lot 😸